how-to/README.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Please also add an info banner to the main readme, saying something like "The wiki is being redesigned, there may be broken links and inconsistent design. Sorry for inconvenience."